lib/log.js

```diff
@@ -36,6 +40,10 @@ var levels = {
 
 _.each(levels, function(level, levelName) {
   Logger.prototype[levelName] = function() {
+    if (this.level === 'silent') {
```

Comment:
This is how the silent functionality is implemented. If the level is "silent", it immediately returns instead of logging.